storey/flow.py

```diff
@@ -132,11 +163,28 @@ def _set_error(self, element):
 
 
 class FlowController:
+    """Used to emit events into the associated flow, terminate the flow, and await the flow's termination.
+    To be used from a synchronous context.
+    """
+
     def __init__(self, emit_fn, await_termination_fn):
         self._emit_fn = emit_fn
         self._await_termination_fn = await_termination_fn
 
     def emit(self, element, key=None, event_time=None, return_awaitable_result=False):
+        """Emits an event into the associated flow.
+
+        :param element: The event data, or payload.
```

Comment:
worth specifying that you can also pass an Event object here

Reply:
Yeah, need to document it *everywhere* I guess...